src/components/Registration/Registration.jsx

```diff
@@ -0,0 +1,50 @@
+import axios from 'axios';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styles from './Registration.module.css';
+
+import Button from '../../common/Button/Button';
+import Input from '../../common/Input/Input';
+
+const Registration = () => {
+  const submitRegister = async (event) => {
+    event.preventDefault();
+    const res = await axios.post('http://localhost:4000/register', {
+      name: event.target.elements.name.value,
+      email: event.target.elements.email.value,
+      password: event.target.elements.password.value,
```

Comment:
Let's decompose `event.target.elements`. It will look like this in your function args:
```suggestion
     const submitRegister = async ({ target: { elements } }) => {
```